src/StakeVault.sol

```diff
@@ -184,4 +225,8 @@ contract StakeVault is Ownable {
             STAKING_TOKEN.transfer(_destination, STAKING_TOKEN.balanceOf(address(this)));
         }
     }
+
+    function _stakeManagerImplementationTrusted() internal pure virtual returns (bool) {
+        return true;
+    }
```

Comment:
This will change in future commits, once #73 lands. Because then we can check if the stake manager's implementation address has changed (at which point the manager is no longer trusted unless explicitly trusted)